src/components/Input/index.tsx

```diff
@@ -97,30 +157,84 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>(
       block,
       defaultValue,
       id,
+      onClickCopy = null,
```

Comment:
the default seems unnecessary. we can also destructure these fields here too so that they don't need to be accessed through `props`
```suggestion
      onClickCopy,
      type,
      value,
      'data-testid': testId,
```